.github/workflows/docs-linting.yml

```diff
@@ -0,0 +1,35 @@
+name: Documentation Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  # linting:
```

Comment:
ℹ️ Note: this is commented specifically because I'm choosing to split up the Markdown linting PR. If you feel it necessary to delete in this PR, that's fine and I'll do so. 👍